README.md

```diff
@@ -4,11 +4,16 @@ TODO: Document the principles and API
 
 ## Setup
 
-An S3 bucket is required, to store the uploaded files. This bucket should have the minimum possible permissions; list objects, put object, delete object.
+An S3 bucket is required, to store the uploaded files. This bucket
+should have the minimum possible permissions; list objects, put object,
+delete object.
 
-`scripts/setup/` contains scripts and files to create an S3 bucket, an IAM user, and apply the appropriate security policy.
+`scripts/setup/` contains scripts and files to create an S3 bucket, an
```

Comment:
We should link to https://github.com/ministryofjustice/mojfile-s3-bucket-setup instead.